sql/moz-fx-data-shared-prod/firefox_ios_derived/funnel_v1/query.sql

```diff
@@ -0,0 +1,102 @@
+WITH views_data AS (
+  SELECT
+    `date`,
+    territory AS country_name,
+    SUM(impressions_unique_device) AS views,
+  FROM
+    app_store.firefox_app_store_territory_source_type_report
+  WHERE
+    `date` >= '2022-01-01'
+  GROUP BY
+    `date`,
+    country_name
+),
+downloads_data AS (
+  SELECT
+    `date`,
+    territory AS country_name,
+    SUM(total_downloads) AS downloads,
+  FROM
+    app_store.firefox_downloads_territory_source_type_report
+  WHERE
+    `date` >= '2022-01-01'
+    AND source_type <> 'Institutional Purchase'
+  GROUP BY
+    `date`,
+    country_name
+),
+store_stats AS (
+  SELECT
+    DATE(`date`) AS `date`,
+    code AS country,
+    views,
+    downloads,
+  FROM
+    views_data
+  JOIN
+    downloads_data
+  USING
+    (`date`, country_name)
+  JOIN
+    static.country_codes_v1
+  ON
+    country_name = name
+),
+activations AS (
+  SELECT
+    first_seen_date AS `date`,
+    country,
+    SUM(new_profile) AS new_profiles,
+    SUM(activated) AS activations,
+  FROM
+    firefox_ios.new_profile_activation
+  WHERE
+    submission_date >= '2022-01-01'
+    AND first_seen_date >= '2022-01-01'
+    AND NOT (
+      app_display_version = '107.2'
+      AND submission_date >= '2023-02-01'
+    ) -- incident filter  #TODO: is there an additional ticket or bug we can reference here?
```

Comment:
I'm not familiar with this incident or why this filter is here, maybe ping vanessa when she's back

Reply:
This is something I kept from the original query @bani wrote. Copying over my previous comment regarding this in case it was missed.

> @bani Do we have a relevant bug or issue we could link here so that if needed we can easily find more context in the future?

Reply:
Here is the doc that explains the issue: https://docs.google.com/document/d/1LewXE8sfz3BM397Bi4nBz1-_WThTW79IY95KzqA0ZnY/edit

I cleaned up the data specifically for some requests we had for some meetings, but I'm open to removing the filter for consistency with other metrics.

Reply:
I don't have enough context around this to make a decision. @fbertsch @bani please let me know if this should stay or if it should be removed from this piece of logic.

Reply:
my preference is to keep it, as it's an easy filter to have more reliable and consistent data within the funnel for a time period people are actively looking at the data.

Reply:
Ok, happy to keep it in that case.